### PR TITLE
Brings back Kelotane

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request

This PR brings the chemical kelotane back into the game inside burn kits in pill bottles, kelotane is a great burn healer

## Why It's Good For The Game

brings a great chemical back

## Changelog
:cl:
add: We have bought pills the chemical Kelotane out of a nearby starsector's Ketamine factory, the long gone chemical is now available in burn medkits!
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
You should probably add a good 'Why' to your PR, because otherwise it's going to be obliterated for the same reason it was removed in the first place

**Comment by TiviPlus** [comment]
Time green didn't look too closely, did he

**Comment by Time-Green** [comment]
> Time green didn't look too closely, did he

"brings a great chemical back" is not a good 'Why' considering the controversial nature of healing chems

**Comment by trollbreeder** [comment]
It's actually ketamine and i think this is an april fools PR

**Comment by TiviPlus** [comment]
Yep, Time-Green didnt look :honk:

**Comment by Time-Green** [comment]
> Yep, Time-Green didnt look :honk:

What are you talking about? This is my first comment

**Comment by TiviPlus** [comment]
to_chat(M, "<span class='userdanger'>You don't think that was Kelotane, wait, is that... Ketamine?</span>")

**Comment by Time-Green** [comment]
> ```
> to_chat(M, "<span class='userdanger'>You don't think that was Kelotane, wait, is that... Ketamine?</span>")
> ```

Are you hallucinating?

File: code/game/objects/items/storage/firstaid.dm [hunk 1]
```diff
@@ -452,6 +453,14 @@
 	for(var/i in 1 to 5)
 		new /obj/item/reagent_containers/pill/neurine(src)
 
+/obj/item/storage/pill_bottle/kelotane
+	name = "bottle of kelotane pills"
+	desc = "Contains pills to treat burn damage. The label seems kinda scratched off"
+
+/obj/item/storage/pill_bottle/neurine/PopulateContents()
```
Comment (Tlaltecuhtli) [inline code/game/objects/items/storage/firstaid.dm]: copypaste error

**Comment by spookydonut** [comment]
Obvious webedit PR.

**Comment by wesoda25** [comment]
This is like a fever dream

**Pull request closed without merging**